### PR TITLE
Loader for cumulative effects hotspots

### PR DESCRIPTION
**Pull request by PhilippeTh**
RAQDPS-FW Cumulative Effects Hotspots loader, data from 2013 to 2020 already added using the script.

### CODE REVIEW

**Comment by Dukestep** [comment]
Tests failing for unrelated reasons. I do notice some long lines in there though, @PhilippeTh. Can you run `flake8`?

File: msc_pygeoapi/loader/cumulative_effects_hs.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+            yield action
+
+    def load_data(self, filepath):
+        """
+        loads data from event to target
+        :returns: `bool` of status result
+        """
+
+        self.filepath = Path(filepath)
```
Comment (tomkralidis) [inline msc_pygeoapi/loader/cumulative_effects_hs.py:134]: This should be set in the initializer (`None`) for safeguarding.

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Comment by PhilippeTh** [comment]
@tomkralidis Done!

**Comment by tomkralidis** [comment]
@PhilippeTh can you resolve the  merge conflicts?

**Comment by tomkralidis** [comment]
@PhilippeTh yes failures are unrelated. Is this PR ready for another review then?

**Comment by PhilippeTh** [comment]
@tomkralidis It is!

**Pull request merged**